docs/source/index.rst

```diff
@@ -62,6 +62,19 @@ termination using the message id.
 
   abort(message_id)
 
+Abort a task using a custom abort_ttl value
+------------
+
+.. code-block::
+
+  @dramatiq.actor
+  def my_long_running_task(): ...
```

Comment:
Long running task does not trigger the issue. It is instead delay in processing. An example with `task.send_with_options(delay=60*60*1000)` would make more sense.